llama_stack/apis/inference/inference.py

```diff
@@ -237,7 +237,7 @@ async def completion(
     @webmethod(route="/inference/chat_completion")
     async def chat_completion(
         self,
-        model: str,
+        model_id: str,
```

Comment:
`llama-stack-apps` examples also needs to be updated: https://github.com/meta-llama/llama-stack-apps/blob/0dc9c42fb42bf21d35e6d231afc4e0360a9eac61/examples/inference/client.py#L46-L49

Reply:
https://llama-stack.readthedocs.io/en/latest/getting_started/index.html#chat-completion-test --> also these docs

Reply:
done.

Reply:
llama-stack-apps and llama-stack-client-python also needs to be updated to reflect the `model` -> `model_id` change.  

https://github.com/meta-llama/llama-stack-apps/blob/0dc9c42fb42bf21d35e6d231afc4e0360a9eac61/examples/inference/client.py#L46-L49